Kickstarter-iOS/Views/PledgeCTAContainerView.swift

```diff
@@ -169,27 +164,10 @@ final class PledgeCTAContainerView: UIView {
 
 private func adaptableStackViewStyle(_ isAccessibilityCategory: Bool) -> (StackViewStyle) {
   return { (stackView: UIStackView) in
-    let axis: NSLayoutConstraint.Axis = (isAccessibilityCategory ? .vertical : .horizontal)
     let spacing: CGFloat = (isAccessibilityCategory ? Styles.grid(1) : 0)
 
     return stackView
-      |> \.axis .~ axis
+      |> \.axis .~ NSLayoutConstraint.Axis.horizontal
```

Comment:
Was the compiler giving you trouble here?  😄 I think that `|> \.axis .~ .horizontal` is more readable and more used on the codebase. Also, we have more styling going on inside the `bindStyles()` function. To keep consistency and avoid doing the same thing in multiple places, I would either move this code to the bindStyles function (not recommended) or do a small refactor bringing the other styles to the bottom. (recommended, since that's how we are approaching styling).

Reply:
It's not that the compiler would give us trouble but we've been seeing a lot of warnings lately related to Xcode's type checker...therefore at some point we've decided that for styles it would make more sense to be more explicit.

I'll make the styling more consistent by extracting `bindStyles` here.